uportal-war/src/main/webapp/media/skins/respondr/common/less/navigation.less

```diff
@@ -43,6 +45,47 @@
         }
     }
 
+   .logout-launcher,.notification-icon,.portal-greeting,.login-launcher{
```

Comment:
Yes in a perfect world, but in responsive web design, you will have a lot of different situations according to css breackpoints but also according to regions, with regions, the possibilities are multiplied...